jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -54,8 +53,10 @@ def buildExclusions = [
   [ /^smartos17/,                     anyType,     gte(12) ],
   [ /^smartos18/,                     anyType,     lt(12)  ],
 
-  // PPC BE ------------------------------------------------
+  // PPC ---------------------------------------------------
   [ /^ppcbe-ubuntu/,                  anyType,     gte(8)  ],
+  [ /^ppcle-ubuntu/,                  anyType,     gte(13) ],
+  [ /^centos7-ppcle/,                 anyType,     lt(12)  ],
```

Comment:
I don't understand the categorization, or have strong feelings about it. Some of the categories are OS, some are machine architecture, and its not clear which organization to prefer. Note that before this change, all the `centos*` were under Linux (arm, intel, and ppc). After, the centos-ppc moved to "PPC", but lthe centos-arm is still in "Linux", instead of in "ARM". Seems like it gains and loses consistency in equal amounts.